### ISSUE TITLE
Print list of repos not i .projects.gws

### ISSUE THREAD

**Issue by JohanSpaedtke** [emlun/gws2#10]
If multiple persons are updating the same `.projects.gws` it can be nice to keep the repos checked out on your machine up to date with what's specified in `.projects.gws`. 

If gws could figure out which repos present that were missing from `.projects.gws` and print them in a nice list that would be very useful. 

Not sure about the name
`gws untracked`, `gws missing` maybee?

Another option would be if `gws status` simply printed a status like "local only"

**Comment by emlun** [comment]
This would probably be a new option or two for the `check` command, no?

**Comment by JohanSpaedtke** [comment]
Is the `check` command something that already exists? Can't find it in my version... Or do you mean this thing should be part of a new command called `check`?

Otherwise just adding a new status like "local only" to `status` feels sort of ok or?

**Comment by emlun** [comment]
No, `check` exists in the original gws but isn't yet implemented in gws2. :)
